docs/source/user/jitclass.rst

```diff
@@ -162,6 +168,9 @@ must be unboxed or boxed between Python objects and native representation.
 Values encapsulated by a jitclass does not get boxed into Python object when
 the jitclass instance is handed to the interpreter.  It is during attribute
 access to the field values that they are boxed.
+Calling static methods as class attributes is only supported outside of the
+class definition (i.e. you can't call ``Bag.add()`` from within another method
+of ``Bag``).
```

Comment:
The error message when calling a static method as a class attribute from another method doesn't appear to be tested.

Reply:
Should be done in 2acbd48